docs/plugins/google-cloud.md

```diff
@@ -152,4 +152,4 @@ This option will force Genkit to export telemetry and log data when running in t
 
 ## Test your integration
 
-When configuring the plugin, use `forceDevExport: true` to enable telemetry export for local runs. Navigate to the Google Cloud Logs, Metrics, or Trace Explorer to view telemetry. Alternatively, navigate to the Firebase AI Monitoring dashboard for an AI-idiomatic view of telemetry.
+When configuring the plugin, use `forceDevExport: true` to enable telemetry export for local runs. Navigate to the Google Cloud Logs, Metrics, or Trace Explorer to view telemetry. Alternatively, navigate to the [Firebase AI Monitoring dashboard](https://console.firebase.google.com/project/_/genai_monitoring) for an AI-idiomatic view of telemetry.
```

Comment:
Customers still won't be able to just see it in the console until we move out of private preview - want to link to the form to sign up?